app/Services/CalculateJobTimeService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use DateTime;
+
+class CalculateJobTimeService
+{
+    public function calculate(string $startedAt, string $completedAt): int
+    {
+        $startedAt = new DateTime($startedAt);
+        $completedAt = new DateTime($completedAt);
+
+        $interval = $startedAt->diff($completedAt);
+
+        $seconds = ($interval->days * 24 * 60 * 60) + 
+                ($interval->h * 60 * 60) + 
+                ($interval->i * 60) + 
+                $interval->s;
+
+        $minutes = (int)ceil($seconds / 60);
```

Comment:
Let's use [Carbon](https://carbon.nesbot.com/docs/), he has all of of methods for things like that

Reply:
changed service to use Carbon